inst/extdata/content_about.md

```diff
@@ -0,0 +1,58 @@
+## About `shinyFuse`
+
+`ShinyFuse` is an application for exploration of fusion output derived from [annoFuse](https://github.com/d3b-center/annoFuse).
+
+This app has two main sections, each with specific functionality, described here in brief.
+
+#### About FusionExplorer
+
+FusionExplorer is an interactive way to explore the putative oncogenic fusion results from annoFuse. 
+
+**Required input:** `PutativeDriverAnnoFuse.tsv`
+
+**Features:** 
+
+_Filtering_: Select a column and enter one or more attributes on which to filter the rows. <br>
```

Comment:
```suggestion
_Filtering_: On the left, select columns to display.
1. If filtering a non-numeric column, select a level from the dropdown on which to filter rows.
2. If filtering a numeric column, use the sliding scale or enter numeric logic according to the following (Eg: to filter for a fusion `SpanningFragCount` greater than 10, write `10 ... max` (number, space, ellipsis, space, number) in the filter box, where `max` is the highest number on the sliding scale). <br>
```
The preview looks a bit weird here, so just be sure this looks right.

Reply:
Will do!

Reply:
Some fixes are in, now looks like this - seems correct to me!
<img width="872" alt="image" src="https://user-images.githubusercontent.com/7447478/93139473-1cd4bd00-f6e1-11ea-8676-659440d8b556.png">